screens/Favorite.tsx

```diff
@@ -11,15 +11,15 @@ export default function Favorite({navigation}) {
   return (
     <View style={{flex: 1, alignItems: 'center', justifyContent: 'center'}}>
       <Text>Hi From User FavoriteScreen</Text>
-      <View style={{flexDirection: 'row', justifyContent: 'space-between'}}>
+      {/* <View style={{flexDirection: 'row', justifyContent: 'space-between'}}>
         <Button title="Pilots" onPress={() => navigation.navigate('Pilots')} />
         <Button title="New" onPress={() => navigation.navigate('New')} />
         <Button
           title="Favorite"
           onPress={() => navigation.navigate('Favorite')}
         />
         <Button title="Logout" onPress={() => navigation.navigate('Login')} />
-      </View>
+      </View> */}
     </View>
   );
 }
```

Comment:
Why is this file's extension is `tsx`? Doesn't looks like you are using typescript in this project, so you can switch to `jsx`

Reply:
<img width="430" alt="Screen Shot 2022-03-04 at 11 19 07 PM" src="https://user-images.githubusercontent.com/53321967/156873001-d4a7beb8-ab01-4139-841a-958a2b4226f2.png">

I think it is part of the default settings of the application when we created it. by changing them to jsx I get this error.

I could replace them for components but didn't work for the files in the screens folder

Reply:
Apparently, metro doesn't have a config to resolve `jsx` so you need to add one. This config tells metro(the js server you run with `npm run start`) what kind of file you would like metro to resolve. The reason it's crashing is that metro doesn't know what to do with `jsx` file. But again, `tsx` is a file extension for typescript, so it just confuses other developers that typescript is used in the file even though it's not!

Well, React Native team probably think there's no one creating a new project with javascript anymore😄

```suggestion
  resolver: {
    sourceExts: [
      'js', // note this has to be defined first or you get an error
      'json',
      'jsx',
      'mjs',
      // required because the react-native cli ignores `resolverMainFields`
      'ts',
      'tsx',
    ],
  },
```

source: https://stackoverflow.com/a/70409871